### ISSUE TITLE
Need an RPC command to get a RAW transaction minimum fee

### ISSUE THREAD

**Issue by MithrilMan** [chronokings/huntercoin#102]
Need an RPC command to get the minimum fee required to create a raw transaction
this way, using daemon would be easier to generate transaction without putting insufficient (or too many) fees

i think min fee should be available as RPC command because it's a daemon responsability to know that 

P.S.
don't know if it's already available, but sendrawtransaction should check if the minimum fee has been used in the transaction

**Comment by domob1812** [comment]
What are you trying to do?  The raw transaction interface is designed to be "raw", i. e., `sendrawtransaction` should just send the transaction and not do any higher-level "bitching around about the fee".  (If the fee is too low to be relayed at all, it won't accept the tx anyway.)

For sending transactions and having the daemon handle the internals (which includes determining an appropriate fee), you should use the higher-level interface like `sendtoaddress` and `name_update`.

**Comment by MithrilMan** [comment]
problem is that i would like to create a transaction on behalf of some address i don't have in the wallet (but i know priv key, just not taken from wallet)
i want to create a raw transaction, sign it then send it from an hot wallet/daemon that doesn't hold such address

Try to explain better: think about my bounty system i talked about on forum, what i want is a service that connect to a running daemon, and has a wallet with a readonly address (loaded using importwallet) that is the address i use to get bounty request people send
I want to use the same service, to pay back to bounty killers when conditions are satisfied, so i need to generate a transaction using that readonly address, then sign using the private key i get from other sources (not wallet), maybe even on another pc, and than use the running daemon to send the signed transation to peers

so basically i can't use sendtoaddress because i need to pick the transaction inputs from the outputs sent to the bounty address and that wallet don't hold anyway the private key so it can't sign the transaction, so i have to build it, but then the problem is that to build such transaction without knowing the fee isn't easy (afaik)

Let me know If i haven't explained thing properly

**Comment by domob1812** [comment]
Ok, the usecase is clear now.  How do Bitcoin applications solve this?  It actually shouldn't be too hard to compute the fee.  Or just attach, say, 0.05 HUC to be on the safe side (which isn't really anything at all).  If `sendrawtransaction` fails, you will know that you have to increase the fee.
